### DISCUSSION TITLE
discussions on discord

### DISCUSSION THREAD

**Discussion by nnako** [freeplane/freeplane#411] [General]
Hi,

yesterday during our 2nd user conference, @rexelbartolome introduced a relatively new discussion channel for Freeplane. It is hosted on the [discord platform](https://discord.com/).

Now, in order for the user community to be able to find the right places for questions / discussions / answers, I'd like to discuss best practices, here. According to my own knowledge, there are the following locations for discussion-related activities:

----

**USER-LEVEL DISCUSSIONS / GROOVY-SCRIPTING**
everything related to working with the Freeplane mindmap editor as well as how to use / create scripts for automating / extending the Freeplane editor UI
```
- Open Discussion space on SourceForge      - deprecated
  (https://sourceforge.net/p/freeplane/discussion/758437)

- Discussions on GitHub                     - used for new posts
  (https://github.com/freeplane/freeplane/discussions)
```

----

**DEVELOPER-LEVEL DISCUSSIONS**
everything related directly to the Java code base, implementation interfaces, code structure, ... which might not be of major interest for the average Freeplane user but could also be a source of inspiration for users who want to extend their development skills.
```
- Freeplane Developers Forum                - not reachable anymore
  (http://freeplane-developer.996965.n3.nabble.com)
  
- Freeplane Discord Channel                 - new? access granted by Rexel?
  (https://discord.com/channels/956691317742788618/956691739517800538)

- Developer Mailing List                    - still in use? still valid?
  (freeplane-developer@lists.sourceforge.net)
```

----

Will this be ok for everyone? Any objections? Will the discord location be **the** location for development-related stuff? Any other suggestions? Anything missed, here?

Thanks

**Comment by rexelbartolome** [comment]
I prefer using Discord mainly because that's what I use for my friends and most of my work/clients. And I made an unofficial "Freeplane" server so that I could talk to Predrag and Dimitry in a more casual environment. 

Threads and text posts could become tedious especially when you're only waiting for the response of one or two people. Also, some of my inquiries could be more conveniently answered through a call and accompanying screenshare which Dimitry did for me multiple times during the development of the icons.

So I think having access to this kind of  environment between the contributors could help make development faster and more agile, compared to the standard Discussions forum.

Everyone's free to join the Discord server, just click this link: https://discord.gg/VA4PFeZVxU
